lib/rubocop/cop/rspec/capybara/match_style.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      module Capybara
+        # Checks for usage of deprecated style methods.
+        #
+        # @example when using `assert_style`
+        #   # bad
+        #   expect do
+        #     page.find(:css, '#first').assert_style(display: 'block')
+        #   end.not_to raise_error
+        #
+        #   # good
+        #   expect do
+        #     page.find(:css, '#first').assert_matches_style(display: 'block')
+        #   end.not_to raise_error
```

Comment:
There are some leftovers of `expect { }.not_to raise_error` here and in specs.

Reply:
Oops, I updated it. Thank you so much.